README.md

```diff
@@ -16,6 +16,7 @@ Request IDs can be specified by prefacing your command with `@int`, for example
 | L       | Control the debug LED       | State {H, L} |                        |
 | R       | Read a digital pin          | Pin Number   |                        |
 | S       | Control a servo             | Servo Number | Width                  |
+| T       | Read a raw ultrasound timing | Trigger Pin  | Echo Pin               |
```

Comment:
description could be better e.g. Read a triggered pulse delay

Reply:
fixed in 06541c9bfca36547e606e81bb8becaf70a3df4d0